README.md

```diff
@@ -24,3 +66,8 @@ To run the unit tests in isolation:
 ```
 npm run unit
 ```
+
+When done you want to test a new branch or test files have been updated, clean up before setting up again:
```

Comment:
```suggestion
When done, if you want to test a new branch or test files have been updated, clean up before setting up again:
```

Reply:
When you just change js code, you don't have to takedown. If you or someone else changes an interview (a .yml file), you need to run takedown and then setup again.

Reply:
```suggestion
If you or someone else changes the interview code in `./docassemble`, you have to clean up the old data on the server before running `setup` again:
```